client/dive-common/components/Viewer.vue

```diff
@@ -507,12 +523,15 @@ export default defineComponent({
           </v-alert>
           <v-progress-circular
             v-else
-            indeterminate
+            :indeterminate="progress.progress < 0"
+            :value="Math.round(progress.progress / progress.total * 100)"
```

Comment:
can we `rotate=-90` this.  I find the default position weird.

Reply:
Sure.